drivers/user_permission_report/app/controllers/user_permission_report/warehouse_reports/users_controller.rb

```diff
@@ -0,0 +1,24 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module UserPermissionReport::WarehouseReports
+  class UsersController < ApplicationController
+    before_action :set_user
+    helper_method :group_associations
+
+    def show
+      render layout: 'ajax_modal_rails/content'
```

Comment:
This becomes unnecessary if you include `include AjaxModalRails::Controller` above.

Reply:
✅ Cool. I could tell that I was missing some magic.